domains/infoblock.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "My Personal Website",
+    "repo": "https://github.com/InfoBlock/Website",
+    "owner": {
+        "username": "InfoBlock",
+        "email": "infoblock.official@gmail.com",
+        "discord": "InfoBlock#4792"
+    },
+    "record": {
+        "CNAME": "infoblock.github.io"
```

Comment:
404?

Reply:
i cant reach the website, it says 404

Reply:
> i cant reach the website, it says 404

I can reach it tho https://infoblock.github.io/Website
Another domain is currently linked to it - https://infoblock.gq